arm-network/2016-06-01/network.json

```diff
@@ -6515,14 +6601,143 @@
       ],
       "description": "NetworkSecurityGroup resource"
     },
+	"EffectiveNetworkSecurityGroup": {
+      "properties": {
+        "networkSecurityGroup": {
+	      "$ref": "#/definitions/SubResource",
+          "description": "Gets the id of network security group that is applied"
```

Comment:
Also sibiling properties of the $ref will be ignored. The Description, should be in the reference definition